web3/_utils/validation.py

```diff
@@ -152,10 +155,23 @@ def validate_abi_value(abi_type: TypeStr, value: Any) -> None:
     )
 
 
+def is_ordinary_string(value: Any) -> bool:
```

Comment:
Maybe this would be more accurately named: `is_non_address_string` or something with the same intent?